bigtable-hbase-1.x-replication/src/main/java/com/google/cloud/bigtable/hbase/replication/BigtableWALEntryImpl.java

```diff
@@ -0,0 +1,24 @@
+package com.google.cloud.bigtable.hbase.replication;
+
+import com.google.cloud.bigtable.hbase.replication.adapters.BigtableWALEntry;
+import org.apache.hadoop.hbase.Cell;
+import org.apache.hadoop.hbase.wal.WAL;
+
+import java.util.ArrayList;
+/*
+BigtableWALEntryImpl provides BigWALEntry class from hbase1x. WAL.Entry
+ */
+public class BigtableWALEntryImpl  {
```

Comment:
This class can be in the core package. You don't need to depend on the WAL. you can create a constructor  liek

```
BigtableWalEntry(String tableName, List<Cell> cells, long writeTimeInMillis)
```